src/applications/appeals/10182/tests/containers/ConfirmationPage.unit.spec.jsx

```diff
@@ -7,7 +7,8 @@ import formConfig from '../../config/form';
 import initialData from '../schema/initialData';
 
 import ConfirmationPage from '../../containers/ConfirmationPage';
-import { SELECTED, FORMAT_READABLE } from '../../constants';
+import { FORMAT_READABLE } from '../../constants';
```

Comment:
All the date format constants have been copied into the shared constants folder

Reply:
Woot woot